lib/http-routes/api/create-call.js

```diff
@@ -127,16 +127,41 @@ router.post('/', async(req, res) => {
 
     /* launch outdial */
     let sdp, sipLogger;
+    let dualEp;
+    let localSdp = ep.local.sdp;
+
+    if (req.body.dual_streams) {
+      dualEp = await ms.createEndpoint();
+      const sdpParts = dualEp.local.sdp.split('m=');
```

Comment:
I would prefer if we use the sdp-transform package for manipulating SDPs. we are already using it in utils/siprec-utils.js as an example

Reply:
start using sdp-transform